tools/package_gcsfuse_docker/Dockerfile

```diff
@@ -36,8 +36,8 @@ ARG DEBFULLNAME="GCSFuse Team"
 
 # Build Arg for building through a particular branch/commit. By default, it uses
 # the tag corresponding to passed GCSFUSE VERSION
-ARG BRANCH_NAME="v${GCSFUSE_VERSION}"
-RUN git checkout "${BRANCH_NAME}"
+# The below hard-coded change is purely for testing, and will be reverted before merge.
+RUN git checkout gargnitin-wip-golang-1.20.5
```

Comment:
Please remove this to get approval

Reply:
Yes, this is a temp change in a separate temp commit for running louhi pipeline for now. Will remove it once Louhi pipeline runs and succeeds.